server/repository/src/migrations/views/mod.rs

```diff
@@ -384,6 +385,27 @@ pub(crate) fn rebuild_views(connection: &StorageConnection) -> anyhow::Result<()
       CREATE VIEW requisitions_in_period AS
           SELECT 'n/a' as id, program_id, period_id, store_id, order_type, type, count(*) as count FROM requisition WHERE order_type IS NOT NULL
             GROUP BY 1,2,3,4,5,6;   
+
+  CREATE VIEW vaccination_card AS
+    SELECT 
+      vcd.id || pe.id AS id,
+      vcd.id as vaccine_course_dose_id, 
+      vcd.label, 
+      vcd.min_interval_days, 
+      vcd.min_age, 
+      vc.id as vaccine_course_id, 
+      v.id as vaccination_id, 
+      v.vaccination_date, 
+      v.given, 
+      v.stock_line_id, 
+      pe.id as program_enrolment_id
+    FROM vaccine_course_dose vcd 
+    INNER JOIN vaccine_course vc 
```

Comment:
Specifying `INNER` not strictly necessary (you didn't for the `JOIN program_enrolment` below)